packages/blockchain-api/src/blockscout.ts

```diff
@@ -51,6 +45,8 @@ export interface BlockscoutTransaction {
 }
 
 export class BlockscoutAPI extends RESTDataSource {
+  tokenAddressMapping: { [key: string]: string } | undefined
```

Comment:
Any reason why we cache these values here in this class and also in the utils file?

Reply:
Will just cache these in this class

Reply:
I cached these both places to avoid passing them back and forth. I realize this isn't the cleanest solution, but the alternatives of:
- Only caching them in the class, and passing them (sometimes undefined) in as parameters to the getter function is a weird pattern
- Only caching them in the function, and needing to call and store the addresses in the class each time they're used
Seemed less desirable